test/conversion_test.go

```diff
@@ -115,7 +115,6 @@ spec:
     volumeMounts:
     - name: messages
       mountPath: /messages
-    script: echo test
```

Comment:
why was this removed?

Reply:
This was previously invalid syntax which aimed to test as many fields as possible in Task but not tested for sidecar.